gfc/financial_companion/templates/pages/individual_transaction.html

```diff
@@ -1,51 +1,36 @@
-                  <h4>Transaction Info:</h4>
-                </div>
-                <ul class="list-group list-group-flush">
-                  <li class="list-group-item"><p><strong>Transaction Description: </strong>{{transaction.description}}</p></li>
-                  <li class="list-group-item"><p><strong>Transaction Category: </strong>{{transaction.category.name.title}}</p></li>
-                  <li class="list-group-item"><p><strong>Transaction Amount: </strong>{{transaction.amount}}</p></li>
-                  <li class="list-group-item"><p><strong>Transaction Currency: </strong>{{transaction.currency.upper}}</p></li>
-                  <li class="list-group-item"><p><strong>Sender Account: </strong>{{transaction.sender_account.name.title}}</p></li>
-                  <li class="list-group-item"><p><strong>Reciver Account: </strong>{{transaction.receiver_account.name.title}}</p></li>
-                  <li class="list-group-item"><p><strong>Time of Transaction: </strong>{{transaction.time_of_transaction}}</p></li>
-                </ul>
-              </div>
-
-            </div>
-          </div>
-        </div>
-      </div>
-    </div>
-  </div>
-</div>
+{% extends "partials/view_models/view_model.html" %}
+
+{% block object_name %}{{transaction.title.title}}{% endblock %}
+{% block model_name %}Transaction{% endblock %}
+
+{% block page_header_model %}Transaction{% endblock %}
+{% block page_header_name %}{{transaction.title.title}}{% endblock %}
+
+{% block card_title %}Transaction Information:{% endblock %}
+
+{% block edit_delete_buttons %}
+  {% url 'edit_transaction' transaction.id as edit_transaction %}
+  {% url 'delete_transaction' transaction.id as delete_transaction %}
+  {% include "partials/view_models/edit_and_delete_models_buttons.html" with edit_URL=edit_transaction delete_URL=delete_transaction model_name="transaction"%}
+{% endblock %}
+
+{% block confirmation_model_name %}transaction{% endblock %}
+
+{% block table_content %}
+  {% if transaction.description %}
+    <li class="list-group-item"><strong>Description:  </strong>{{transaction.description}}</li>
+  {% endif %}
+  {% if transaction.category.name %}
+    <li class="list-group-item"><strong>Category:  </strong>{{transaction.category.name.title}}</li>
+  {% endif %}
+  <li class="list-group-item"><strong>Amount:  </strong>{{transaction.currency.upper}} {{transaction.amount}}</li>
+  <li class="list-group-item"><strong>Sender Account:  </strong>{{transaction.sender_account.name.title}}</li>
+  <li class="list-group-item"><strong>Receiver Account:  </strong>{{transaction.receiver_account.name.title}}</li>
+  <li class="list-group-item"><strong>Time of Transaction:  </strong>{{transaction.time_of_transaction}}</li>
 {% endblock %}
+
+{% block additional_items %}{% include "partials/view_models/model_photo.html" with picture=transaction.image name=transaction.title.title%}{% endblock %}
+
+{% block view_model_objects_URL %}{% url 'dashboard' %}{% endblock %}
```

Comment:
Please make it so that it redirects you to the view transactions page instead of dashboard. Thank you

Reply:
I do apologise for this frivolous error and have resolved this issue.